data/modules/TradeShips/Core.lua

```diff
@@ -0,0 +1,159 @@
+-- Copyright © 2008-2021 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local Game = require 'Game'
+local Serializer = require 'Serializer'
+
+-- this module is responsible for storing global variables, logging, serialization
+local Core = {
+
+	-- CONSTANTS / SETTINGS
+	AU = 149598000000.0, -- meters
+
+	MAX_ROUTE_FLOW = 3, -- flow to the most popular station in the system, ships/hour
+	MAX_BUSY = 0.4, -- maximum station load
+	MAX_SHIPS = 500, -- ~ maximim ships in open space (limitation for performance reasons)
+	MIN_STATION_DOCKING_TIME = 2, -- hours, minimum average time of the ship's stay at the station
+
+	WAIT_FOR_NEXT_UNDOCK = 600, -- seconds
+	MINIMUM_NEAR_SYSTEMS = 10,
+	LOG_SIZE = 2000, -- records in log
+
+	-- serializable table
+	ships = nil,
+	--[[
+	ships:
+		<key> - object returned from Space:SpawnShip*
+		ship_name - of this ship type; string
+		starport - at which this ship intends to dock; SpaceStation object
+		dest_time - arrival time from hyperspace; number as Game.kjtime
+		dest_path - for hyperspace; SystemPath object, may have body index
+		from_path - for hyperspace; SystemPath object
+		delay - indicates waiting for a Timer to give next action; number as Game.time
+		status - of this ship; string, one of:
+			hyperspace - yet to arrive
+			hyperspace_out - has departed
+			inbound - in system and given AIDockWith order
+			docked - currently docked or un/docking
+			outbound - heading away from starport before hyperspacing
+			fleeing - has been attacked and is trying to get away
+			(currently still just following whatever AI order it had)
+			cowering - docked after having been attacked, waiting for
+			           attacker to go away
+			orbit - was unable to dock, heading to or waiting in orbit
+		attacker - what this was last attacked by; Body object
+		chance - used to determine what action to take when attacked; number
```

Comment:
With large comments like this, please prefer putting them outside of the table initializer braces and reformatting them to read something like:
```lua
--[[
    Property: Core.ships
    ...
--]]
Core.ships = nil
```

It helps with readability and generally makes the file look nicer when doc comments are structured consistently.

Reply:
Done!